### PR TITLE
add Heiman matter Motion and door sensor

### PR DESCRIPTION
**Pull request by greens**

### CODE REVIEW

**Comment by Leo2442926161** [comment]
Hi @lelandblue , thanks for reminding, signed.

File: drivers/SmartThings/matter-sensor/fingerprints.yml [hunk 1]
```diff
@@ -33,6 +33,17 @@ matterManufacturer:
     vendorId: 0x1021
     productId: 0x0001
     deviceProfileName: motion-contact-battery
+  #Heiman
+  - id: "Heiman Motion"
+    deviceLabel: Heiman Motion
+    vendorId: 0x120B
+    productId: 0x1001
+    deviceProfileName: matter-motion-battery-illuminance
+  - id: "Heiman DoorAndWindow"
```
Comment (hubTab) [inline drivers/SmartThings/matter-sensor/fingerprints.yml:42]: In accordance with the rest of the fingerprints, should this id be `id: "Heiman/DoorAndWindow"` (see [id: "Eve/DoorAndWindow](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1026/files#diff-d421b34194f8b655303df53e5d9d5d991a51cef4fb3882b9f9858aae830074b7L14)", [id: "TUO/ContactDoorAndWindow"](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1026/files#diff-d421b34194f8b655303df53e5d9d5d991a51cef4fb3882b9f9858aae830074b7L25))

Comment (greens) [reply drivers/SmartThings/matter-sensor/fingerprints.yml:42]: we don't really have hard and fast rules about what the id for fingerprints should be, and they're not user-facing

**Review by hubTab — APPROVED** [review]

**Comment by Leo2442926161** [comment]
Thanks a lot for your help @greens @hubTab @lelandblue

**Comment by Leo2442926161** [comment]
Hi @z-michel , could you help to review the PR, thanks.

**Comment by lelandblue** [comment]
Yes I brought that up the partner success manager and they are getting back to me. We will most likely close the other PR with the four devices in it.

**Pull request closed without merging**